DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -2571,6 +2571,10 @@
 		BDADBDCD2BD2BC5700421B9B /* vpn-light-mode.json in Resources */ = {isa = PBXBuildFile; fileRef = BD384AC72BBC821100EF3735 /* vpn-light-mode.json */; };
 		BDE981D92BBD10D600645880 /* vpn-dark-mode.json in Resources */ = {isa = PBXBuildFile; fileRef = BD384AC82BBC821100EF3735 /* vpn-dark-mode.json */; };
 		BDE981DA2BBD10D600645880 /* vpn-light-mode.json in Resources */ = {isa = PBXBuildFile; fileRef = BD384AC72BBC821100EF3735 /* vpn-light-mode.json */; };
+		C11198312C898AB500F0272C /* FreemiumDBPFirstProfileSavedNotifier.swift in Sources */ = {isa = PBXBuildFile; fileRef = C11198302C898AB500F0272C /* FreemiumDBPFirstProfileSavedNotifier.swift */; };
```

Comment:
Here we have a type named `…DBP…`. So, this PR includes things named like this AND like `…PIR…`. This is known, and relates to a Tech Design discussion re: renaming PIR to DBP. However, rather than rename all things here from PIR to DBP, I will instead open a PR to the feature branch before the merge to main. At that point I will rename where needed and ensure we are consistent in our usage of DBP